test/integration/membersController.test.js

```diff
@@ -0,0 +1,33 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+const addUser = require('../utils/addUser')
+
+chai.use(chaiHttp)
+
+describe('Members', function () {
+  before(async function () {
+    await addUser()
+  })
+
+  describe('GET /members', function () {
+    it('Get all the members in the database', function (done) {
```

Comment:
Can you also add a test when there are no members in DB?

Reply:
We had not handled the case when there are no members in the DB. Made the change here: https://github.com/Real-Dev-Squad/website-backend/pull/167
Please review